### ISSUE TITLE
Make slow COUNT tip clearer

### ISSUE THREAD

**Issue by lrweck** [pgMustard/pgMustard#43]
On the plan https://app.pgmustard.com/#/explore/dde14238-a7df-48eb-859d-bd21a9132adc?highlighted=10

The suggestion is about count(*) being slow, and it is rated as 5 star. However the node highlighted amounted to less than a millisecond of a 2 second query.

Can this be improved?

**Comment by mchristofides** [comment]
Thanks for the message, and good question. 

Your final Aggregate operation is small, but the operations below it are also required to calculate the count(*). Combined, these are taking a good chunk of the overall time, and pgMustard is using this overall subtree time to estimate the time-saving potential of using a different approach. 

The easiest way to see the total time of a subtree in pgMustard is currently to collapse it with the "-" button to the left of an operation (the Aggregate in this case).

Most of our tips are node-specific, so tips like this that take more information into account have been tricky to communicate well. So far we've tried to put them on the most sensible node. One idea is to introduce query level tips, but that is also difficult to communicate. A more specific idea for this would be to include the overall time and percentage for the count(*) subtree in the tips description/text.

It'd be great to hear your thoughts, and of course do let me know if I've misunderstood anything.

**Comment by lrweck** [comment]
One thing I've seen used to express this is the concept of inclusive and exclusive time spent. Maybe you have already considered it. Exclusive times being the time spent on the node itself, and inclusive is the sum of the times spent up to the current node.

**Comment by mchristofides** [comment]
Yes spot on. We find the exclusive times most useful most of the time, hence calculating and displaying those prominently in pgMustard, but this is a great example of the inclusive time (and percentage of total) being valuable too. Do you remember other cases where you've found inclusive times valuable?

We'll have a think about how to make the inclusive time clearer here, thanks again.

**Comment by lrweck** [comment]
I can't really think of a situation out of the top of my head, but maybe sort is a example...

**Comment by mchristofides** [comment]
Well if you remember or come across others please feel free to send them over!

I've looked over our tips and don't think any of the others we provide use inclusive time yet, but we'll keep this in mind when changing them or adding new ones.

**Comment by mchristofides** [comment]
@lrweck thanks again for submitting this issue. We have decided to improve the tip wording to make it clearer.

We added the following: "The COUNT itself is often quite fast, but the child operations to read and process the rows to be counted can take much longer."

Thanks again. Do let us know if you come across anything else unclear or incorrect! 

Here it is in context:
![count](https://user-images.githubusercontent.com/10532911/101364266-46455500-389a-11eb-9f97-da6ee5b9aca0.jpg)
